client/src/Components/Stars.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Popover } from 'react-bootstrap';
+import { OverlayTrigger } from 'react-bootstrap';
+import { Table } from 'react-bootstrap';
+
+class Stars extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  getStars(stars) {
+    let starsArray = ['☆','☆','☆','☆','☆'];
```

Comment:
Are you directly display these text stars? Are you going to replace them with SVGs or images?